generatebundlefile/ecr_helper.go

```diff
@@ -106,7 +106,7 @@ func ImageTagFilter(details []ImageDetailsBothECR, version string) []ImageDetail
 	var filteredDetails []ImageDetailsBothECR
 	for _, detail := range details {
 		for _, tag := range detail.ImageTags {
```

Comment:
Our bundles are like `v1-24-latest` is that going to be a problem with this?

Reply:
No it wouldn't be a problem with that, this is only for promoting helm chart, so it's not using the k8s version in this logic at all.

I have it working locally, just working on adding some tests right now.